client/package-notes.md

```diff
@@ -0,0 +1,29 @@
+# Gobra IDE Build System
```

Comment:
For whom is this file? For us developers or can IDE users also see this?

Reply:
Just for us. As the README will be visible to IDE users, I've had to create a different file for that